create docs rn for june (#3666)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/07-June-2023/product-docs-jun.md

---
title: "June 2023 product docs updates"
description: "June 2023: The Product docs team merged 132 PRs, made various updates to dbt Cloud and Core, such as the Deploy sidebar, Supported platforms page, added a landing page on the References section, added an ADO example to the CI/CD guide, and more"
sidebar_label: "Update: Product docs changes"
tags: [June-2023, product-docs]
date: 2023-07-04
sidebar_position: 10
---

> Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 17 new community contributors to docs.getdbt.com!

Reviewer: curious, should we list the new contributors here (@ mention them) after the last sentence in this paragraph? i think it might be a good shoutout for them ✨

Author: i think that's a wonderful idea @nghi-ly !!!!!! jerco does this as well for the core releases ✨ i think if we have a month where there are a lot of contributors, we can prob also link to the search to not type all the names.

Reviewer: 🙌 

didn't know that about core releases -- cool!

> Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 17 new community contributors to docs.getdbt.com &mdash; ✨ aaronbini, sjaureguimodo, aranke, eiof, tlochner95, mani-dbt, iamtodor, monilondo, vrfn, raginjason, AndrewRTsao, MitchellBarker, ajaythomas, smitsrr, leoguyaux, GideonShils, michaelmherrera!

Reviewer: question: wondering, should we @ mention them? with the rss integration and the community channels, the slack notification could be neat 🤷‍♀️

Author: omg i initially added the `@` and then stopped. i can add it but not sure if their git names are the same as their slack names right? or whether they're in slack at all! i wouldn't want to go and search for names manually in slack as that can lead to a lot of work down the road

Reviewer: yeah, that's true

Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/) in June:

## ☁ Cloud projects

> - We clarified the nuances of [CI/CD and Slim CI](/docs/deploy/continuous-integration), updated the [Scheduler content](/docs/deploy/job-scheduler), added two new pages for the job settings and run visibility, moved the project state page to the [Syntax page](/reference/node-selection/syntax), and provided a landing page for [Deploying with Cloud](/docs/deploy/dbt-cloud-job) to help readers navigate the content better.

Reviewer: ```suggestion
- We clarified the nuances of [CI and Slim CI jobs](/docs/deploy/continuous-integration), updated the [Scheduler content](/docs/deploy/job-scheduler), added two new pages for the job settings and run visibility, moved the project state page to the [Syntax page](/reference/node-selection/syntax), and provided a landing page for [Deploying with Cloud](/docs/deploy/dbt-cloud-job) to help readers navigate the content better.
```

- We reformatted the [Supported data platforms page](/docs/supported-data-platforms) by adding dbt Cloud to the page, splitting it into multiple pages, using cards to display verified adapters, and moving the [Warehouse setup pages](/docs/core/connect-data-platform/about-core-connections) to the Docs section. 

> - We launched a new [Lint and Format page](/docs/cloud/dbt-cloud-ide/lint-format), which highlights the awesome new dbt Cloud IDE linting/formatting function.

Reviewer: sugg sentence case vs title case (per style guide)

```suggestion
- We launched a new [Lint and format page](/docs/cloud/dbt-cloud-ide/lint-format), which highlights the awesome new dbt Cloud IDE linting/formatting function.
```

- We enabled a connection between [dbt Cloud release notes](/docs/dbt-versions/dbt-cloud-release-notes) and the dbt Slack community. This means new dbt Cloud release notes are automatically sent to the slack community [#dbt-cloud channel](https://getdbt.slack.com/archives/CMZ2V0X8V) via RSS feed, keeping users up to date with changes that may affect them. 

> - We’ve added two new docs links in the dbt Cloud Job settings user interface (UI). This addition will provide users with more guidance and help them experience success in dbt cloud when setting up a job: [job commands](/docs/deploy/job-commands) and [job triggers](/docs/deploy/job-triggers).    

Reviewer 2: ```suggestion
- We’ve added two new docs links in the dbt Cloud Job settings user interface (UI). This will provide additional guidance and help users succeed when setting up a dbt Cloud job: [job commands](/docs/deploy/job-commands) and [job triggers](/docs/deploy/job-triggers).    
```

> - We added information related to the newly created [IT license](/docs/cloud/manage-access/about-user-acces), available for Team and Enterprise plans. 

Author: ```suggestion
- We added information related to the newly created [IT license](/docs/cloud/manage-access/about-user-access#license-based-access-control), available for Team and Enterprise plans. 
```

> - We added a new [Supported browser page](/docs/cloud/about-cloud/browsers) which lists the recommended browsers to use in dbt Cloud.

Reviewer 2: ```suggestion
- We added a new [Supported browser page](/docs/cloud/about-cloud/browsers), which lists the recommended browsers for dbt Cloud.
```
